### ISSUE TITLE
TypeScript - Find all references not working for default exports

### ISSUE THREAD

**Issue by dbaeumer** [Microsoft/TypeScript#7024]
_From @use-strict on February 5, 2016 13:47_

"Find all references" feature doesn't report any usages for a class that is exported as default. See example below:

```typescript
// B.ts
export default class B { // <-- "find all references" on B returns no usages
    test() {
    }
}
// A.ts
import B from "./B";

let b = new B();
b.test();
```
VSCode version:
```
Version 0.10.6
Commit a80232bbcfe8a5cdad1ebc98638673d9dcb02458
Date 2015-12-19T11:26:42.359Z
Shell 0.34.5
Renderer 45.0.2454.85
Node 4.1.1
```

_Copied from original issue: Microsoft/vscode#2733_

**Comment by aluanhaddad** [comment]
:+1:  I would love to see this improvement.
